build1.sh

```diff
@@ -11,8 +11,9 @@ cmake \
     -DWITH_RUNTIME_STACKTRACE=yes \
     -DWITH_LSP=no \
     -DCMAKE_PREFIX_PATH="$CMAKE_PREFIX_PATH_LFORTRAN;$CONDA_PREFIX" \
-    -DCMAKE_INSTALL_PREFIX=`pwd`/inst \
+    -DCMAKE_INSTALL_PREFIX=$(pwd)/inst \
     -DCMAKE_INSTALL_LIBDIR=share/lfortran/lib \
     -DCMAKE_EXPORT_COMPILE_COMMANDS=yes \
+-G Ninja \
     .
-cmake --build . -j16 --target install
+ninja -j16 install
```

Comment:
I think ninja runs in parallel by default, correctly using the number of CPUs on your computer:
```suggestion
ninja install
```

Furthermore, let's use the `cmake` to invoke it, like this:
```
cmake --build . --parallel --target install
```
I am not sure if the `--parallel` is needed.

Reply:
We can use either `--parallel N` or `-jN`. Using them without `N` will lead to infinite jobs with `make`, with `ninja`, we use the max thread count for the CPU.

#### make
```console
-j [N], --jobs[=N]          Allow N jobs at once; infinite jobs with no arg.
```

#### ninja
```console
-j N     run N jobs in parallel (0 means infinity) [default=8 on this system]
```

Reply:
On my system, the `--parallel` option causes the memory to fill rapidly and eventually hang the system.